test/mjml-props.test.tsx

```diff
@@ -0,0 +1,143 @@
+import React from "react";
+
+import * as mjmlComponents from "../src";
+import { renderToMjml } from "../src/utils/renderToMjml";
+
+/**
+ * Test focused on prop values. The component used for testing the prop
+ * is arbitrary.
+ */
+describe("mjml components prop values", () => {
+  it("should render component with dangerouslySetInnerHTML prop", () => {
+    expect(
+      renderToMjml(
+        <mjmlComponents.MjmlRaw
+          // eslint-disable-next-line react/no-danger
+          dangerouslySetInnerHTML={{ __html: "<div>hello World</div>" }}
+        />
+      )
+    ).toBe(`<mj-raw><div>hello World</div></mj-raw>`);
+  });
+
+  it("incorrect prop value should throw TS error", () => {
+    expect(
+      renderToMjml(
+        <mjmlComponents.MjmlRaw
+          // @ts-expect-error prop should be invalid for test purposes
+          iAmNotAProp="random"
+        />
+      )
+    ).toBe(`<mj-raw i-am-not-a-prop="random"></mj-raw>`);
+  });
+
+  it("fullWidth and inline props convert to string", () => {
+    expect(renderToMjml(<mjmlComponents.MjmlSection fullWidth />)).toBe(
+      `<mj-section full-width="full-width"></mj-section>`
+    );
+    expect(renderToMjml(<mjmlComponents.MjmlStyle inline />)).toBe(
+      `<mj-style inline="inline"></mj-style>`
+    );
+  });
+
+  it("enum prop type accepts all enum values", () => {
+    const { MjmlTable } = mjmlComponents;
+    expect(renderToMjml(<MjmlTable align="left" />)).toBe(
+      `<mj-table align="left"></mj-table>`
+    );
+    expect(renderToMjml(<MjmlTable align="right" />)).toBe(
+      `<mj-table align="right"></mj-table>`
+    );
+    expect(renderToMjml(<MjmlTable align="center" />)).toBe(
+      `<mj-table align="center"></mj-table>`
+    );
+
+    // Test with invalid props. Code should still render but typescript should
+    // throw an error
+    // @ts-expect-error invalid align prop for test purposes
```

Comment:
I haven't used `@ts-expect-error` before. Have you tested that both having an accepted prop and not having this comment cause the test to fail?